internal/frontend/default-frontend.go

```diff
@@ -486,9 +482,10 @@ func parseFlagDefinition(line string) (string, string, string, string, string) {
 	return name, short, description, flagType, defaultValue
 }
 
-func (self *defaultFrontend) getCmdEnvContext(envVars []string, consents []string) []string {
+func (self *defaultFrontend) getCmdEnvContext(cmd command.Command, envVars []string, consents []string) []string {
 	vars := append([]string{}, envVars...)
 
+	/* append environment variables that requires consent */
```

Comment:
```suggestion
	/* append environment variables that require consent */
```
OPT

Reply:
Resolved in commit [c0eed5b](https://github.com/criteo/command-launcher/pull/149/commits/c0eed5b237e1256211c066fda3ce74fc7853f76c)